build.sbt

```diff
@@ -49,10 +49,48 @@ val commonSettings: immutable.Seq[Def.Setting[_]] = List(
     specsScalaCheck,
     specsMock
   ),
-  assembly / assemblyMergeStrategy := {
+  ThisBuild / assemblyMergeStrategy := {
     case "META-INF/MANIFEST.MF" => MergeStrategy.discard
-    case "META-INF/org/apache/logging/log4j/core/config/plugins/Log4j2Plugins.dat" =>
-      new MergeFilesStrategy
+    case PathList(ps @ _*) if ps.last equalsIgnoreCase "Log4j2Plugins.dat" =>
```

Comment:
You may have explained this in our snyk session, but I am not sure why we change the implementation of merge strategy here?  The previous one does not work?

Reply:
yeah, I was getting an error with instantiating the MergeFilesStrategy class.